config/kubernetes/sig-release/teams.yaml

```diff
@@ -27,103 +28,97 @@ teams:
     maintainers:
     - cblecker # ContribEx
     - fejta # Testing
-    - idvoretskyi # PM
-    - mrbobbytables # 1.15 Enhancements Shadow
+    - mrbobbytables # 1.17 Enhancements Lead
     - nikhita # ContribEx
     - spiffxp # Testing
     members:
     - abgworrall # GCP
-    - alejandrox1 # 1.15 CI Signal
+    - adisky # OpenStack
```

Comment:
Is there any particular reason we are still providing access to old SIG-<cloud-provider>-leads now that they've been folded into SIG Cloud Provider?

Reply:
@nikhita -- there's still provider code in k/k, so those contributors will likely still need milestone maintainers privs.